### ISSUE TITLE
Local volume adjustment would be a nice feature :)

### ISSUE THREAD

**Issue by Natenom** [acomminos/Plumble#160]

**Comment by gsauthof** [comment]
yes - currently, the volume up/down buttons on a Nexus 5x phone don't have any effect on the actual Plumble playback volume

**Comment by Natenom** [comment]
I meant the ability to change the volume of every user on the server. Like Mumble does in the development for the upcoming 1.3...
